### ISSUE TITLE
Image Size and Loss function

### ISSUE THREAD

**Issue by 53RT** [nyukat/breast_density_classifier#6]
Hi,

thanks for publishing your code and congratulation for your great work.
There are two things I would like to know but couldn't find them in the breast density paper nor in the Bi-Rads paper.

- What is the typical image size in your dataset before cropping?
- Which loss function have you used in your training?

**Comment by kjgeras** [comment]
- I don't remember the exact number, it would be roughly 20% bigger on each side. 
- cross_entropy, i.e. simply -log y_correct.

**Comment by 53RT** [comment]
thank you for the instant answer
